bootstrap/src/lib/util.ts

```diff
@@ -176,7 +176,7 @@ export const convertUnits = (coin: string, amount: string, unit?: string) => {
   switch (coin.toLowerCase()) {
     case 'btc':
       // default satoshi
-      return new BN(amount).multipliedBy(10 ** 8).toString(10)
+      return new Decimal(amount).mul(10 ** 8).toString()
```

Comment:
Not sure how useful this function actually is, the way it's currently set up. Let me check...